Makefile

```diff
@@ -0,0 +1,5 @@
+run:
+		python main.py
+
+test: 
+		make run & sleep 5 && python -m unittest -v test_main.py
```

Comment:
Mmm why run it from here?
You can load main.py functions from the test runner.
Just call it before the test